README.md

```diff
@@ -9,5 +9,4 @@ For more information regarding Launch extension development, please visit our [d
 ## Development Setup
 1. Install [node.js](https://nodejs.org/). Make sure you have at least [npm v4.0](https://docs.npmjs.com/getting-started/installing-node#updating-npm) installed.
```

Comment:
`npx` is available starting with `npm@5.2.0`(https://github.com/npm/npm/releases/tag/v5.2.0).

Reply:
Good catch. Updated with the text we use on https://github.com/Adobe-Marketing-Cloud/reactor-sandbox